docs/heasarc/heasarc.rst

```diff
@@ -160,7 +160,7 @@ It is also possible to select time range:
                            NAME                           RA     DEC   ... ISSUE_FLAG             SEARCH_OFFSET_           
                                                          deg     deg   ...                                                 
     -------------------------------------------------- -------- ------ ... ---------- -------------------------------------
-    3C273                                              187.2473 2.0362 ...          0 2.077 (187.2779220931,2.0523864229)\n
+    3C273                                              187.2473 2.0362 ...          0 2.077 (187.2779220936,2.0523864234)\n
```

Comment:
doctests are run in a way for rst files of stopping at the first failure. So once one fix has been made, pls keep going, rerun the tests on that file until all failures are fixed. If the numerical values/order of rows are indeterministic, or too much version dependent, add an `IGNORE_OUTPUT`.

(I see the next one on L266).

Reply:
Whoops, sorry about that. I've just gotten too used to another project with really broken up tests. I've made the fixes for the rest of the failing examples in heasarc.rst in my latest commit.

Reply:
No worries, this is one of the particularities of doctesting. And as for the numerical differences, we'll opt into ignoring these outputs if/when they keep changing, but I rather try to keep them in the comparisions first.